#global testerror "Err0128"

func toto() {}

#test
{
    discard toto()
}
